x-pack/test/functional/services/ml/test_resources.ts

```diff
@@ -54,27 +54,40 @@ export function MachineLearningTestResourcesProvider(
       await kibanaServer.uiSettings.unset('hideAnnouncements');
     },
 
-    async savedObjectExistsById(id: string, objectType: SavedObjectType): Promise<boolean> {
-      const response = await supertest.get(`/api/saved_objects/${objectType}/${id}`);
+    async savedObjectExistsById(
```

Comment:
All the changes in this file are to pass the space name around for the functions used by this test